src/main/java/com/github/manolo8/darkbot/gui/players/PlayerEditor.java

```diff
@@ -35,17 +41,38 @@ public PlayerEditor() {
         add(tagEditor = new PlayerTagManager(this), "grow");
 
         playerInfoList = new JList<>(playersModel = new DefaultListModel<>());
-        playerInfoList.setCellRenderer(new PlayerRenderer());
+        playerInfoList.setCellRenderer(renderer = new PlayerRenderer(playerInfoList));
 
         add(new JScrollPane(playerInfoList,
                 JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED, JScrollPane.HORIZONTAL_SCROLLBAR_NEVER), "span, grow");
+
+        playerInfoList.addListSelectionListener((event) -> {
+            if (event.getValueIsAdjusting()) return;
+            int min = Math.max(event.getFirstIndex(), main.config.PLAYER_INFOS.size());
```

Comment:
```suggestion
            int min = event.getFirstIndex();
```
We can't assume the first N entries (player info size) won't be temporal players, because you could be running a filtered search in which case they wouldn't count for that. They're already checked a bit lower down so it's not an issue